apps/console/src/features/roles/constants/role-constants.ts

```diff
@@ -74,6 +74,16 @@ export class RoleConstants {
         "CONSOLE",
         "MY_ACCOUNT"
     ];
+
+    /**
+     * filter query for audience type application.
```

Comment:
```suggestion
     * Filter query for audience type application.
```
We usually follow the standard of starting comments with Capital and ending with a full-stop. Please check other places as well.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4280/commits/31b88ad29860876b65b8843de775b5b7a096cd5e